aws/v14.2.0/release.yaml

```diff
@@ -0,0 +1,68 @@
+# Generated with:
+# devctl release create --name 14.2.0 --provider aws --base 14.1.1 --component aws-operator@10.3.0 --component cert-operator@1.0.1 --component cluster-operator@3.7.1 --overwrite --app cert-manager@2.7.1 --app cluster-autoscaler@1.19.3 --app chart-operator@2.14.0 --app external-dns@2.3.1 --app metrics-server@1.3.0 --app net-exporter@1.10.1 --component app-operator@4.4.0 --component containerlinux@2765.2.3
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v14.2.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.6.1
+  - componentVersion: 1.3.1
+    name: cert-manager
+    version: 2.7.1
+  - name: chart-operator
+    version: 2.14.0
+  - name: cluster-autoscaler
+    version: 1.19.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.4.1
+  - componentVersion: 0.7.6
+    name: external-dns
+    version: 2.3.1
+  - componentVersion: 3.6.0
+    name: kiam
+    version: 1.7.1
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.1
+  - componentVersion: 0.4.1
+    name: metrics-server
+    version: 1.3.0
+  - name: net-exporter
+    version: 1.10.1
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.2
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: false
```

Comment:
releaseOperatorDeploy is now set to false. What is the impact of this change? The other operators have the property set to true.